Makefile

```diff
@@ -45,3 +45,6 @@ test-v:
 
 get-token:
 	python3 zeton_django/scripts/get_token.py --u opiekun1 --p opiekun1 | jq ".access"
+
+dump-data-points:
```

Comment:
Can't this be done in more generic way?
Also - this would be a helpful addition to `zeton_django/README.md`.

Reply:
Definitely - I think I made the decision too quickly, under the influence of excitement that a nice session had aroused in me. ;)